crates/rome_js_formatter/tests/prettier_tests.rs

```diff
@@ -321,7 +329,54 @@ impl DiffReport {
 
             writeln!(report, "```").unwrap();
         }
-
         write("report.md", report).unwrap();
     }
+
+    fn report_metric(&self) {
+        let mut report = String::new();
+        let mut state = self.state.lock();
+        state.sort_by_key(|(name, ..)| *name);
+        let mut sum_of_per_compatibility_file = 0_f64;
+        let mut total_line_of_rome = 0;
+        let mut total_matched_line_of_rome = 0;
+        for (file_name, rome, prettier) in state.iter() {
+            writeln!(report, "# {}", file_name).unwrap();
+            let rome_lines = rome.lines().count();
+
+            let matched_lines = diff_lines(Algorithm::default(), prettier, rome)
+                .iter()
+                .filter(|(tag, _)| matches!(tag, ChangeTag::Equal))
+                .fold(0, |acc, (_, insert_lines)| {
+                    acc + insert_lines.lines().count()
+                });
+            let compatibility_per_file = matched_lines as f64 / rome_lines as f64;
```

Comment:
The definition of the metric is to compute the matching lines over the larger of the two. This is to avoid a 100% match even if the prettier file contains e.g. an additional newline at the end of the file
```suggestion
            let compatibility_per_file = matched_lines as f64 / rome_lines.max(prettier_lines) as f64;
```